src/ui/components/Events/Events.tsx

```diff
@@ -38,7 +41,12 @@ function Events({ currentTime, events, executionPoint, seek }: PropsFromRedux) {
           <div key={e.point}>
             <CurrentTimeLine isActive={currentEventIndex === i} />
             <div className="px-1.5">
-              <Event onSeek={onSeek} event={e} {...{ currentTime, executionPoint }} />
+              <Event
+                onSeek={onSeek}
+                event={e}
+                {...{ currentTime, executionPoint }}
```

Comment:
I still don't know how I feel about this style, but if we're going this route you should include `onSeek`

Reply:
Not a fan anymore — I'm undoing it as I go!